test/doc/algo/any_of.cpp

```diff
@@ -0,0 +1,29 @@
+#include <eve/module/core.hpp>
+#include <eve/algo.hpp>
+#include <iostream>
+#include <vector>
+#include "detail/print.hpp"
+
+int main()
+{
+  std::vector<float> r1(5), r2(5);
+  auto vr1 = eve::views::iota_with_step(0.5f, 2.0f, 5);
+  eve::algo::copy(vr1, r1);
+  auto vr2 = eve::views::iota(-3.0, 5);
```

Comment:
There is also an algo `iota`. That's not going to be faster now but I bet we can make it on gcc.

I'd prefer if you used that.

Reply:
Please? JUst no need to copy.

Reply:
Fixed